presto-clp/src/main/java/com/facebook/presto/plugin/clp/ClpFilterToKqlConverter.java

```diff
@@ -205,6 +208,71 @@ private String getVariableName(VariableReferenceExpression variable, Map<Variabl
         return ((ClpColumnHandle) context.get(variable)).getOriginalColumnName();
     }
 
+    /**
+     * Rewrites CLP UDFs (e.g., CLP_GET_*) in a RowExpression tree into VariableReferenceExpressions,
+     * enabling them to be used as pushdown filters.
+     * <p></p>
+     * Traverses the expression tree recursively, replacing supported CLP UDFs with uniquely named
+     * variables and tracking these variables in the assignments and context. If the CLP UDF takes
+     * a constant string argument, that string is used as the new variable name. Unsupported
+     * argument types (non-constants) or invalid expressions will throw an exception.
+     * <p></p>
+     * Examples:
+     * <ul>
+     *   <li><code>CLP_GET_STRING('field')</code> → <code>field</code> (as a VariableReferenceExpression)</li>
+     *   <li><code>CLP_GET_INT('field')</code> → <code>field</code> (not mapped to a KQL column)</li>
```

Comment:
```suggestion
     *   <li><code>CLP_GET_STRING('field')</code> → <code>field</code> (as a 
     *   {@link VariableReferenceExpression})</li>
     *   <li><code>CLP_GET_INT('field')</code> → <code>field</code> (not mapped to a KQL column)
     *   </li>
```
We'd better make the length to 100 then in the future we can easily format the code